radius.go

```diff
@@ -37,7 +37,7 @@ func start_radius_server(db *sql.DB) {
 			} else {
 				// found the ip
 				res.Code = radius.CodeAccessAccept
-				log.Info("The IP we found for imsi: %s is IP: %s", imsi, ip)
```

Comment:
A different way to do this is with structured logging `log.WithField("imsi", imsi).WithField("ip", ip).Info("Found an IP for IMSI")`. This is really nice when parsing logs, since the fields are kept separately and can even be setup to log into a database with type information maintained. This is okay for now though.